src/routes/auth.route.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const authController = require('../controllers/auth.controller.js');
+const { authMiddleware } = require('../middlewares/authMiddleware.js');
+const catchError = require('../utils/catchError.js');
+
+const authRouter = new express.Router();
+
+authRouter.post('/registration', catchError(authController.register));
+
+authRouter.get(
+  '/activation/:activationToken',
+  catchError(authController.activate),
+);
+authRouter.post('/login', catchError(authController.login));
+authRouter.get('/refresh', catchError(authController.refresh));
+authRouter.post('/logout', catchError(authController.logout));
+authRouter.post('/password-reset', catchError(authController.passwordReset));
+authRouter.post('/reset-password', catchError(authController.resetPassword));
+
+authRouter.put(
+  '/profile',
+  authMiddleware,
```

Comment:
Ensure that `authMiddleware` is correctly implemented to call `next()` and allow the middleware chain to continue. This is crucial for the `/profile` route to function correctly.